apps/native/app/src/utils/get-theme-with-preferences.ts

```diff
@@ -55,9 +55,12 @@ export function getThemeWithPreferences(
 ) {
   // get color scheme from system if "automatic"
   // otherwise from appearanceMode
-  const colorScheme = (
-    appearanceMode === 'automatic' ? systemScheme : appearanceMode
-  ) as ThemeMode
+  // const colorScheme = (
+  //   appearanceMode === 'automatic' ? systemScheme : appearanceMode
+  // ) as ThemeMode
```

Comment:
Kept this intentionally in but commented out for future reference but can remove if that is better 😄